libdrgn/dwarf_info_cache.c

```diff
@@ -16,6 +16,7 @@
 
 DEFINE_HASH_TABLE_FUNCTIONS(dwarf_type_map, hash_pair_ptr_type,
 			    hash_table_scalar_eq)
+DEFINE_VECTOR(type_member_vector, struct drgn_type_member)
```

Comment:
It's more verbose, but I'd prefer to spell out `drgn_type_member_vector` to be explicit.